src/traces/isosurface/attributes.js

```diff
@@ -206,6 +207,10 @@ var attrs = module.exports = overrideAll(extendFlat({
         description: 'Same as `text`.'
     },
     hovertemplate: hovertemplateAttrs(),
+    xhoverformat: hoverformatAttrs.xhoverformat,
+    yhoverformat: hoverformatAttrs.yhoverformat,
+    zhoverformat: hoverformatAttrs.zhoverformat,
```

Comment:
in principle `isosurface` and `volume` also want a `valuehoverformat`

Reply:
but that's a pretty weird edge case - fine to leave as an open issue if it would take more than 10 min to fix.